dataactbroker/handlers/fileHandler.py

```diff
@@ -200,9 +200,11 @@ def finalize(self):
             # Compare user ID with user who submitted job, if no match return 400
             job = self.jobManager.getJobById(jobId)
             submission = self.jobManager.getSubmissionForJob(job)
-            if(submission.user_id != LoginSession.getName(session)):
+            # Check that user's agency matches submission cgac_code or user id of submission
+            userId = LoginSession.getName(session)
+            if(submission.user_id != userId and submission.cgac_code != self.interfaces.userDb.getUserByUID(userId).cgac_code):
```

Comment:
Is this something that people in the `SYS` agency should be able to do? Or folks who are website admins? (honest question--I don't remember the original requirements around this).